docker-swarm-cluster/azuredeploy-parameters.json

```diff
@@ -0,0 +1,68 @@
+﻿{
+    "$schema": "http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "location": {
+            "type": "string",
```

Comment:
types in parameters is not needed. Just a simple parameter name and value will do. 
e.g. here https://github.com/Azure/azure-quickstart-templates/blob/master/101-vm-multiple-data-disk/azuredeploy.parameters.json